src/main/java/gibson/task/Task.java

```diff
@@ -0,0 +1,46 @@
+package gibson.task;
+
+/**
+ * Represents a task with a task string that describes a job to be done
+ */
+public class Task {
+    private String taskString;
+    private boolean done;
```

Comment:
Additionally, though I find the use of booleans more elegant, do you think that perhaps using integer representations (0=not done, 1 = done, 2=in progress, etc.) makes the code more extendable, given we can add more 'status of completion' values?